hedera-node/src/main/java/com/hedera/services/txns/TransitionRunner.java

```diff
@@ -62,7 +63,8 @@
 			TokenCreate,
 			TokenFeeScheduleUpdate,
 			CryptoTransfer,
-			TokenDelete
+			TokenDelete,
```

Comment:
Consistency-wise - we should keep refactored ops grouped together, additionally ordered by life cycle. I think that `TokenDelete `should be moved after `TokenCreate`